src/utils/fileExports.js

```diff
@@ -113,8 +114,8 @@ const formatTransactions = async (transactions, safeAddress) => {
   return transactionData.map((data, index) => {
     if (data.otherSafe === ZERO_ADDRESS) {
       // UBI transaction
-      data.name = 'CirclesUBI';
-      data.paymentNote = 'UBI payout';
+      data.name = translate('ExportStatement.exportDataName');
+      data.paymentNote = translate('ExportStatement.exportPaymentNote');
```

Comment:
now we are adding some text in translations but not all. Maybe then I should add all fields in en.json